test/internet/test-dns-ipv4.js

```diff
@@ -184,3 +185,13 @@ TEST(function test_lookupservice_ip_ipv4(done) {
 
   checkWrap(req);
 });
+
+TEST(function test_lookupservice_ip_ipv4_promise(done) {
+  util.promisify(dns.lookupService)('127.0.0.1', 80)
```

Comment:
@evanlucas Ah, right… added the unhandled rejection handler, like in the other tests